imagemodal/private/view.js

```diff
@@ -13,6 +12,8 @@ function ImageModalView(runtime, element) {
     var curtain = $element.find('.curtain');
     var image = curtain.find('IMG');
     var mask = curtain.find('.mask');
+    var wrapper = $element.find('.wrapper');
+    var buttonFullScreen = wrapper.find('BUTTON.fullscreen');
```

Comment:
Yeah, I've had that habit for ages; I like that the elements "pop" out, as first-class page elements. I don't do it in platform changes (to be consistent w/ existing code), but since I started doing that in this repo, I'm just continuing the pattern now.